### PR TITLE
CVE-2018-14664

### PR DESCRIPTION
**Pull request by lpardoRH**

### CODE REVIEW

File: 2018/14xxx/CVE-2018-14664.json [hunk 1]
```diff
@@ -1,18 +1,76 @@
 {
```
Comment (cve-team) [inline 2018/14xxx/CVE-2018-14664.json]: u'UPSTREAM' is not one of [u'AIXAPAR', u'ALLAIRE', u'APPLE', u'ATSTAKE', u'AUSCERT', u'BEA', u'BID', u'BINDVIEW', u'BUGTRAQ', u'CALDERA', u'CERT', u'CERT-VN', u'CHECKPOINT', u'CIAC', u'CISCO', u'COMPAQ', u'CONECTIVA', u'CONFIRM', u'DEBIAN', u'EEYE', u'ENGARDE', u'ERS', u'EXPLOIT-DB', u'FarmerVenema', u'FEDORA', u'FREEBSD', u'FRSIRT', u'FULLDISC', u'GENTOO', u'HP', u'HPBUG', u'IBM', u'IDEFENSE', u'IMMUNIX', u'ISS', u'JVN', u'JVNDB', u'L0PHT', u'MANDRAKE', u'MANDRIVA', u'MISC', u'MLIST', u'MS', u'MSKB', u'NAI', u'NETBSD', u'NTBUGTRAQ', u'OPENBSD', u'OPENPKG', u'OSVDB', u'OVAL', u'REDHAT', u'SCO', u'SECTRACK', u'SECUNIA', u'SF-INCIDENTS', u'SGI', u'SLACKWARE', u'SREASON', u'SREASONRES', u'SUN', u'SUNALERT', u'SUNBUG', u'SUSE', u'TRUSTIX', u'TURBO', u'UBUNTU', u'VIM', u'VULN-DEV', u'VULNWATCH', u'VUPEN', u'WIN2KSEC', u'XF']

**Comment by cve-team** [comment]
The 'refsource' value should be one of the sources listed at http://cve.mitre.org/data/refs/#sources. References to bug reports will be 'CONFIRM' if the "vendor" acknowledges the vulnerability in some fashion or "MISC" otherwise (say because the vendor has not responded to the bug report).

**Comment by lpardoRH** [comment]
thanks for the clarification, I've submitted an edit for the refsources. Regards

File: 2018/14xxx/CVE-2018-14664.json [hunk 2]
```diff
@@ -1,18 +1,76 @@
...
+                {
+                    "vendor_name": "[UNKNOWN]",
+                    "product": {
+                        "product_data": [
+                            {
+                                "product_name": "foreman",
+                                "version": {
+                                    "version_data": [
+                                        {
+                                            "version_value": "n/a"
```
Comment (cve-team) [inline 2018/14xxx/CVE-2018-14664.json]: The version_value should reflect what's included in the description (in this case it looks like this should be 1.18).

File: 2018/14xxx/CVE-2018-14664.json [hunk 3]
```diff
@@ -1,18 +1,76 @@
...
+                ]
+            }
+        ]
+    },
+    "references": {
+        "reference_data": [
+            {
+                "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2018-14664",
+                "name": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2018-14664",
+                "refsource": "REDHAT"
```
Comment (cve-team) [inline 2018/14xxx/CVE-2018-14664.json]: The "refsource" should be "CONFIRM" instead of "REDHAT" .

**Comment by cve-team** [comment]
Thanks for your submission, this information will be published shortly.

**Pull request merged**